caikit/runtime/http_server.py

```diff
@@ -293,7 +290,7 @@ async def _handler(
         ) -> EventSourceResponse:
             log.debug("In streaming handler for %s", rpc.name)
             request_kwargs = {
-                field: getattr(request, field) for field in request.__fields__
+                field: getattr(request, field) for field in request.model_fields
```

Comment:
could this be just `request_kwargs = dict(request)`?

Reply:
intentionally not call `model_dump` coz we  need flat dict

Reply:
probably? I was just getting rid of a deprecation warning here and I want to limit the scope of this PR

Reply:
cool, either way I already have that change in my rest-train PR 👍

Reply:
hot